python/spacetime/managers/version_graph.py

```diff
@@ -154,4 +168,75 @@ def maintain(self, state_to_ref, merger_function):
         # First divergent.
         self.maintain_nodes(state_to_ref, merger_function, False)
         # The master line.
-        self.maintain_nodes(state_to_ref, merger_function, True)
\ No newline at end of file
+        self.maintain_nodes(state_to_ref, merger_function, True)
+
+
+class VersionGraphProcess(Process):
+
+    def __init__(self):
+        self.graph = Graph()
+        super().__init__()
+        self.manager = Manager()
+        self.result_manager = Manager()
+        self.event_manager = Manager()
+        self.read_write_queue = self.manager.Queue()
+        self.request_list = []
+
+        def create_flask_app(self):
+            app = Flask(__name__)
+            #app.config["DEBUG"] = True
+
+            @app.route('/next', methods=['GET'])
```

Comment:
```
            @app.route('/next', methods=['GET'])
            def graph():
                if self.request_list:
                    # List is empty, and we cannot do any action.
                    return
                req = self.request_list.pop(0)
                if req[0] == "continue_chain":
                    from_version, to_version, package, e1 = req[1:]
                    self.process_continue_chain(from_version, to_version, package, e1)
                elif req[0] == "maintain":
                    state_to_ref, merger_function, e2 = req[1:]
                    self.process_maintain(state_to_ref, merger_function, e2)
                elif req[0] == "get_item":
                    key, result_queue = req[1], req[2]
                    self.process_get_item(key, result_queue)
                return self.graph.convert_to_json()
```

The function should look like this. Dont read the read_write_queue in this function. Read it in the run function.